delfin/drivers/dell_emc/vmax/rest.py

```diff
@@ -453,6 +453,23 @@ def get_system_capacity(self, array, version):
                       {'array': array})
         return capacity_details
 
+    def get_default_srps(self, array, version=U4V_VERSION):
+        """Get the VMax array default SRPs.
+        :param version: the unisphere version
+        :param array: the array serial number
+        :returns: dictionary default_srps
+        """
+        default_fba_srp = None
+        default_ckd_srp = None
+        symmetrix_info = self.get_system_capacity(array, version)
+        if symmetrix_info and symmetrix_info.get('default_fba_srp'):
```

Comment:
This line is not required. Because "symmetrix_info" is nothing but "capacity_details" is not empty (already verified in that methods)
so, 
symmetrix_info.get('default_fba_srp', None)  will serve the all intention. (# 462, 463 can also be removed)

same for line #467

Reply:
thanks, optimized, plz check